web/modules/custom/dpl_event/src/Access/EventSeriesTabAccessCheck.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Drupal\dpl_event\Access;
+
+use Drupal\Core\Access\AccessResult;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Routing\Access\AccessInterface;
+use Drupal\Core\Routing\RouteMatchInterface;
+
+/**
+ * Provides an access checker for the event series tab.
+ *
+ * This access check ensures that the tab for editing event instances
+ * is only available when there are more than one instance in the series.
+ */
+class EventSeriesTabAccessCheck implements AccessInterface {
```

Comment:
I appreciate the code comment.

As I see it the current comment explains what the code does. It could be improved by adding a sentence or two about why this is necessary. This also goes for other DocBlocks.